pfsimulator/parflow_lib/parflow_netcdf.h

```diff
@@ -26,9 +25,13 @@
 #else
 #define MAX_NC_VARS 8192
 #endif
+#include<stdbool.h>
+
+static int ncID, xID, yID, zID, lev1ID, timID;
```

Comment:
Why are static variables being declared in the header file?   This will create local copies in any file that #includes this header file.  Shouldn't these just all bin the .c file.

Reply:
Hi Steve,
You can reformat the code in my branch. I wanted global variables for dimension IDs in netcdf which would be accessible to all related netcdf routines and I do not have to pass too many arguments to functions. In theory they could be moved to individual .c files. I will check how to do it without breaking current working code.